src/building-blocks/FormItem.tsx

```diff
@@ -80,7 +102,7 @@ class FormItem extends Component<IFormFieldProps> {
         <Antd.Form.Item
           className={className}
           {...formItemProps}
```

Comment:
If we changed this to:
```ts
        <Antd.Form.Item
          className={className}
          {...this.formItemProps}
          {...formItemProps}
```
We could have a function that returns `{}` or `{ extra }` and then we wouldn't have to change fieldConafig at all. I think that would be a lot simpler than adding to formItemProps.

The reason I was doing it this way in the original was it was the only way for me to set extra from outside fields-ant, but you could just set it here yourself.